docs-website/src/pages/champions/index.js

```diff
@@ -10,240 +10,106 @@ const championSections = [
-              </p>
-            </>
-          ),
-          social: {
-            },
-        },
-        {
-          name: "Piotr Skrydalewicz",
-          image: "https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/data-practitioners-guild/initials/ps_white.jpg",
-          position: "Data Engineer",
-          bio: (
-            <>
-              <p>
-                Contributed 5 commits in 2022 to the main DataHub Project and Helm repositories, including Stateful Ingestion support for Presto-on-Hive.
-              </p>
-            </>
-          ),
-          social: {
-            linkedin: "https://www.linkedin.com/in/skrydal",
-          },
-          location: "Lodz, Poland"
-        },
-      {
-        name: "Harvey Li",
-        image: "https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/data-practitioners-guild/initials/hl_black.jpg",
-        position: "Lead Data Engineer, Grab",
-        bio: (
-          <>
-            <p>
-            Shared Grab's experience adopting and implementing DataHub during the October 2022 Town Hall and featured in Humans of DataHub.
-            </p>
-          </>
-        ),        
-        social: {
-          linkedin: "https://www.linkedin.com/in/li-haihui",
-          github: "https://github.com/HarveyLeo",
-          },
-        location: "Singapore"
-        },
-      {
-        name: "Fredrik Sannholm",
-        position: "Wolt",
-        image: "https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/data-practitioners-guild/initials/fs_black.jpg",
-        position: "",
-        bio: (
-          <>
-            <p>
-            Drove DataHub adoption at Wolt and featured in Humans of DataHub.           
-            </p>
-          </>
-        ),        
-        social: {
-          },
-        location: "Finland"
         },
-        {
-          name: "Scott Maciej",
-          position: "Optum",
-          image: "https://raw.githubusercontent.com/datahub-project/static-assets/champ-img/imgs/datahub-champions/initials/sm.jpg",
-          bio: (
-            <>
-              <p>
-              Drove DataHub's adaptation and implementation at Optum.
-              </p>
-            </>
-          ),
-          social: {
-            linkedin: "https://www.linkedin.com/in/scottmaciej/",
-            github: "https://github.com/sgm44",
-            web: "https://www.badhabitbeer.com/",
-            },
-          location: "USA"
+        location: "Rio de Janeiro, Brazil"
       },
       {
-        name: "Tim Bossenmaier",
-        image: "https://raw.githubusercontent.com/datahub-project/static-assets/champ-img/imgs/datahub-champions/tim_bossenmaier.jpeg",
-        position: "Data & Software Engineer, Cloudflight",
+        name: "Mike Burke",
+        image: "https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/datahub-champions/initials/mb.jpg",
+        position: ,
```

Comment:
@maggiehays I think this is causing error: 
https://github.com/datahub-project/datahub/actions/runs/12265981022/job/34223189631?pr=12089#step:8:7587

```suggestion
        position: "",
```